src/main/java/org/terasology/module/health/components/DamageSoundComponent.java

```diff
@@ -4,13 +4,18 @@
 
 import com.google.common.collect.Lists;
 import org.terasology.engine.audio.StaticSound;
-import org.terasology.engine.entitySystem.Component;
+import org.terasology.gestalt.entitysystem.component.Component;
 
 import java.util.List;
 
 /**
  * Contains list of damage sounds, one of which is played when entity is damaged.
  */
-public class DamageSoundComponent implements Component {
+public class DamageSoundComponent implements Component<DamageSoundComponent> {
     public List<StaticSound> sounds = Lists.newArrayList();
+
+    @Override
+    public void copy(DamageSoundComponent other) {
+        this.sounds = other.sounds;
```

Comment:
It looks like `StaticSound` is an immutable asset, which means that it should be fine to do a shallow copy of this list, right?